salt/modules/timezone.py

```diff
@@ -410,6 +410,12 @@ def set_hwclock(clock):
         elif clock == 'localtime':
             __salt__['file.sed']('/etc/default/rcS', '^UTC=.*', 'UTC=no')
     elif 'Gentoo' in __grains__['os_family']:
+        if clock not in ('UTC', 'localtime'):
+            raise SaltInvocationError(
+                'Only \'UTC\' and \'localtime\' are allowed'
+            )
+        if clock.__eq__('localtime'):
```

Comment:
Yes, but the ``__eq__()`` method should almost never be invoked directly, so I was curious why this choice was made.